packages/api/bin/serve.js

```diff
@@ -135,7 +135,10 @@ async function checkOrCreateTables(stackName) {
 async function prepareServices(stackName, bucket) {
   setLocalEsVariables(stackName);
   console.log(process.env.ES_HOST);
-  await bootstrapElasticSearch(process.env.ES_HOST, process.env.ES_INDEX);
+  await bootstrapElasticSearch({
```

Comment:
Most of the updates in terms of file volume in this PR are related to the changed/paramterized signature for bootstrapElasticSearch.